host/testdata/matching_simulation_get_partition_config_from_db.yaml

```diff
@@ -0,0 +1,29 @@
+enablearchival: false
+clusterno: 1
+messagingclientconfig:
+  usemock: true
+historyconfig:
+  numhistoryshards: 4
+  numhistoryhosts: 1
+matchingconfig:
+  nummatchinghosts: 4
+  simulationconfig:
```

Comment:
I'd keep the `simulationconfig` section close to the default one as much as possible. Each scenario should differ from each other only in the dimensions it is supposed to highlight.